mathesar_ui/src/stores/table-data/constraints.ts

```diff
@@ -153,17 +125,18 @@ export class ConstraintsDataStore implements Writable<ConstraintsData> {
 
     try {
       this.promise?.cancel();
-      this.promise = this.api.get({
-        limit: 500,
-        ...this.shareConsumer?.getQueryParams(),
-      });
 
-      const response = await this.promise;
+      // TODO_BETA Do we need shareConsumer here? Previously we had been
+      // passing:
+      //
+      // ```
+      // ...this.shareConsumer?.getQueryParams()
+      // ```
+      this.promise = api.constraints.list(this.apiContext).run();
```

Comment:
This will depend on the changes have to make for publicly shared links. The consumer here provides a `shared-link-uuid` query param which used to get validated for each request made to any publicly shared table (& it's related APIs).